examples/mistral/requirements.txt

```diff
@@ -0,0 +1,6 @@
+datasets
+git+https://github.com/intel/neural-compressor.git
+onnxruntime-extensions
+optimum
```

Comment:
should we add a minimum version for optimum here? Just to be safe?

Reply:
will add the 1.14.1 as the minimal requirement